library/globals.inc.php

```diff
@@ -3037,5 +3037,58 @@
         'user_specific_globals' => $USER_SPECIFIC_GLOBALS,
         'user_specific_tabs' => $USER_SPECIFIC_TABS ] );
 }
+  // PQRS (Physician Quality Reporting System)
+  // 2016-02-02 leebc -- Globals configuration section for PQRS
+  // 2016-03-23 leebc -- Added configuration values for registry for XML
 
+  'PQRS' => array(
+
+    'enable_pqrs' => array(
+      xl('Enable Physician Quality Reporting System (PQRS/MIPS)'),
+      'bool',                           // data type
+      '1',                               // default
+      xl('Enable Physician Quality Reporting System (PQRS/MIPS)')
+    ),
+
+    'pqrs_demosystem' => array(
+      xl('This is a PQRS demo system'),
+      'bool',                           // data type
+      '0',                               // default
+      xl('Show demo system "Save/Load database presets" menu')
+    ),
+
+    'pqrs_creator' => array(
+      xl('PQRS Creator'),		// for XML generation
+      'text',                           // data type
+      'FIXME creator FIXME!!!',                     // default
```

Comment:
Yes, if it isn't configured, when you run the xml output, you get a report that has all the unconfigured stuff in it.  Has to be wrong to start with...unless you want me to put some company's registration numbers in there.